docs/source/markdown/options/rootfs.md

```diff
@@ -21,3 +21,12 @@ finishes executing, similar to a tmpfs mount point being unmounted.
 
 Note: On **SELinux** systems, the rootfs needs the correct label, which is by default
 **unconfined_u:object_r:container_file_t:s0**.
+
+    The `idmap` option if specified, create an idmapped mount to the target user
+namespace in the container.
+The idmap option supports a custom mapping that can be different than the user
```

Comment:
Why do you want this?  When would you want the idmapping to be differnt the User Namespace mapping?

Reply:
some users are using it as a way to restrict access to the file system to certain IDs, we had a request for crun: https://github.com/containers/crun/issues/873